statistics-dashboard-service/src/main/java/eu/europeana/statistics/dashboard/service/utils/RequestUtils.java

```diff
@@ -0,0 +1,98 @@
+package eu.europeana.statistics.dashboard.service.utils;
+
+import eu.europeana.statistics.dashboard.common.api.request.StatisticsValueFilter;
+import eu.europeana.statistics.dashboard.common.api.request.StatisticsFilteringRequest;
+import eu.europeana.statistics.dashboard.common.iternal.FieldMongoStatistics;
+import eu.europeana.statistics.dashboard.service.exception.FacetDeclarationFailException;
+import eu.europeana.statistics.dashboard.service.persistence.StatisticsQuery.ValueRange;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class RequestUtils {
```

Comment:
I think this is also a final with a private constructor?

Reply:
I can do that